cypress/support/commands.ts

```diff
@@ -120,3 +129,11 @@ Cypress.Commands.add("leaveEditMode", () => {
   cy.get("[data-cy=leave-edit-mode]").click();
   return cy.get("[data-cy=nurseShiftsTable]", { timeout: 10000 });
 });
+
+export function numberOfWeeksInMonth(month: number, year: number): number {
+  const [missingPrev, missingNext] = calculateMissingFullWeekDays(new ScheduleKey(month, year));
+  const monthLength = daysInMonth(month, year).length;
+  const fullScheduleDays = missingPrev + monthLength + missingNext;
+
+  return Math.ceil(fullScheduleDays / numberOfDaysInWeek);
```

Comment:
tutaj pytanie czy to nie zawsze powinno być podzielne przez 7 i co lepiej zrobić czy zrobić ceil jak robisz, czy rzucić error czy coś innego. Trzeba się zastanowić

Reply:
Tak, zawsze przez 7, tylko nie chciałem dawać po prostu 7 tylko stałą, żeby było jasne o co chodzi. Z tym, że mogłem tę stała nazwać wielkimi literami. Musi być ceil, bo zawsze chcemy pełną ilość tygodni np. przy 39 dniach to musi być 6

Reply:
chodziło mi, że fullScheduleDays nigdy nie może być 39 bo nie jest podzielne przez 7, jak z przodu i z tylu już uzupelniasz dniami do równych tygodni

Reply:
Kurde faktycznie, coś mnie przymroczyło XD Ceil nie przeszkadza, ale jest niepotrzebny i wprowadza zamieszanie